### PR TITLE
fix: remove calico interface when remove nodes

### PR DESCRIPTION
**Pull request by angao**
**What type of PR is this?**
/kind bug
/area network

**What this PR does / why we need it**:
- 移除节点后，移除 calico network device
  - 之前 calico 创建 device 是 `dummy0`，目前版本是 `vxlan.calico`

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @hanxueluo @WangMinjie114 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hanxueluo** [comment]
看看还有没有其它内容残留的？比如 iptables、ipset 之类。

File: roles/network_plugin/bd-calico/tasks/reset.yml [hunk 1]
```diff
@@ -8,13 +8,49 @@
...
+  stat:
+    path: /sys/class/net/tunl0
+  register: tunl0
+
+- name: reset | remove the tunl0 device created by calico
+  command: ip link del tunl0
+  ignore_errors: True
+  when: tunl0.stat.exists
+
+- name: reset | get remaining routes set by vxlan
```
Comment (hanxueluo) [inline roles/network_plugin/bd-calico/tasks/reset.yml:31]: device 删除了，路由不会自动删除?

Comment (angao) [reply roles/network_plugin/bd-calico/tasks/reset.yml:31]: 确保一下吧

**Pull request merged**